near-contract-standards/src/non_fungible_token/core.rs

```diff
@@ -3,6 +3,13 @@ use crate::non_fungible_token::token::{Token, TokenId};
 use near_sdk::json_types::ValidAccountId;
 use near_sdk::PromiseOrValue;
 
+/// Trait used for all non-fungible tokens. The specification for the
+/// [core non-fungible token standard] lays out the reasoning for each method.
+/// It's important to check out [NonFungibleTokenReceiver](crate::non_fungible_token::receiver::NonFungibleTokenReceiver)
+/// and [`crate::non_fungible_token::resolver::NonFungibleTokenResolver`] to
```

Comment:
maybe also worth it to shorten this link as well (you can even add the alias with the link below if it clutters the comment)

Reply:
good catch, pushed up. thanks